saleor/order/tests/test_order_utils.py

```diff
@@ -326,3 +328,41 @@ def test_add_variant_to_order(order, customer_user, variant, site_settings):
     assert line.undiscounted_unit_price == unit_price
     assert line.undiscounted_total_price == total_price
     assert line.tax_rate == tax_rate
+
+
+def test_update_taxes_for_order_line_deleted_variant(order_with_lines):
+    # given
+    order_line = order_with_lines.lines.first()
+    order_line_unchanged_copy = copy.deepcopy(order_line)
+    unit_discount_amount = Decimal("2.00")
+
+    unit_price = TaxedMoney(net=Money("10.23", "USD"), gross=Money("15.80", "USD"))
+    total_price = TaxedMoney(net=Money("30.34", "USD"), gross=Money("36.49", "USD"))
+    tax_rate = Decimal("0.23")
```

Comment:
Where are these values coming from, are they hardcoded? Will the test pass if we make an unrelated change in `order_with_lines` fixture and it would return different values?

Reply:
Those values are different from values in the fixture (they are copied from other tests from `test_order.utils.py`). `update_taxes_for_order_line` method should not update prices in line with those values. So changing anything in fixture should not interfere with the test. The test only checks if no changes in prices were made by `update_taxes_for_order_line` in line when variant is `None`.

Reply:
👍